lib/gui/pages/main/controllers/image-selection.js

```diff
@@ -122,20 +123,22 @@ module.exports = function(
    * ImageSelectionController.selectImageByPath('path/to/image.img');
    */
   this.selectImageByPath = (imagePath) => {
-    imageStream.getImageMetadata(imagePath)
-      .then(this.selectImage)
-      .catch((error) => {
-        const imageError = errors.createUserError({
-          title: 'Error opening image',
-          description: messages.error.openImage({
-            imageBasename: path.basename(imagePath),
-            errorMessage: error.message
-          })
+    $timeout(() => {
```

Comment:
Lol my bad, I don't know what I was doing wrapping a promise in the timeout. I've fixed it now.